webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -4,11 +4,28 @@
 from django.http import HttpResponseBadRequest, Http404
 from django.shortcuts import redirect
 from django.core.urlresolvers import reverse
+from django.shortcuts import render
 
 from . import utils
 from crashstats.base import ga
 
 
+def decorate_bad_request(view):
```

Comment:
Ah, OK, I see what you want to express. ``prettify_bad_request`` or ``better_bad_request`` sound good. 

But to be honest, the fact that we struggle finding a good name for this is another argument in favor of not making this a decorator. :)